app-infrastructure/outputs.tf

```diff
@@ -10,6 +10,10 @@ output "dictionary-ec2-id" {
   value = aws_instance.dictionary-ec2.id
 }
 
-output "hpds-ec2-id" {
-  value = var.env_is_open_access ? aws_instance.open-hpds-ec2[0].id : aws_instance.auth-hpds-ec2[0].id
+output "hpds-ec2-open-id" {
```

Comment:
I would change these to `open-hpds-ec2-id` and `auth-hpds-ec2-id` for consistency.

Reply:
Also, requires relevant updates here https://github.com/hms-dbmi/avillachlab-jenkins/blob/open-picsure-infrastructure/jenkins-docker/jobs/Await%20Initialization/config.xml#L74-L87

Reply:
@bp85 I already included the first change and the second one is something I will fix.

Reply:
https://github.com/hms-dbmi/pic-sure-bdc-infrastructure/blob/open-picsure-infrastructure/app-infrastructure/outputs.tf